wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/button/InternalButton.java

```diff
@@ -1,44 +0,0 @@
-// Copyright (c) FIRST and other WPILib contributors.
-// Open Source Software; you can modify and/or share it under the terms of
-// the WPILib BSD license file in the root directory of this project.
-
-package edu.wpi.first.wpilibj2.command.button;
-
-/**
- * This class is intended to be used within a program. The programmer can manually set its value.
- * Also includes a setting for whether or not it should invert its value.
- *
- * <p>This class is provided by the NewCommands VendorDep
- */
-public class InternalButton extends Button {
```

Comment:
Is it ok to remove this without a deprecation year? Likely not.

Reply:
Restored, no API changes other than the removed `get()` method (which there isn't much we can do about--it's removed at a higher level in the inheritance hierarchy).